app/models/to_do.rb

```diff
@@ -0,0 +1,7 @@
+class ToDo < ActiveRecord::Base
+
+  def as_json
+    { id: id, title: title, is_completed: is_completed }
```

Comment:
This works. As something to think about, we usually use [ActiveModelSerializers](https://github.com/rails-api/active_model_serializers) for a more OO approach or at least [jbuilder](https://github.com/rails/jbuilder) to keep it out of the model.